### DISCUSSION TITLE
Congrats to every one... but we're not done!

### DISCUSSION THREAD

**Discussion by coverbeek** [frc5567/2023-Robot#164] [Team Posts]
Obviously, the results of the weekend are fantastic and I am just as excited as all of you about how it went. We do still have opportunity to improve on our overall system and add significant automation to our working systems. We will need to commit all of the changes made during competition this week and everyone will want to get the latest code once that has been done.

Additionally, we made significant changes to the scouting software and have some room for improvement there. If there are coding team members who would like to work on the scouting software talk to one of the mentors.

Never let a success become an excuse to rest on past accomplishment -- it should instead drive you to strive for greater successes!

**Comment by coverbeek** [comment]
With Danny's approval, I've also invited a couple of my young co-workers to our GitHub as they are interested in possibly contributing as mentors going forward. 
Some of you may have met Thomas Bosler (@TBosler ) when he stopped by our build space. He is an alumnus of FRC Team 4325 RoboRangers (https://www.thebluealliance.com/team/4325/2019). He stopped by our competition for a while on Friday, then headed to St. Joe to watch his team lead the 2nd seeded alliance to victory there!
Jake Haas (@haasjac ) is also a FIRST alum, hailing from FRC Team 3414 Hackbots (https://www.thebluealliance.com/team/3414)
We have at least two other FIRST alums at my office, but I haven't yet lured them to considering helping us out.

**Comment by MikasaSucasa** [comment]
I could not agree more. Many teams haven't even competed yet and I promise you that we have been noticed. Teams will only get better from here on out. I think that much of the foundation has been laid out for us to easily expand upon an autonomous mode and auto-assists. Never did I imagine that we'd be 4th in the state right now, and that is worthy of some serious congrats but know that everyone will be coming at us even harder now.

Awesome job to everyone! Even the little things, like the gamepad, won awards. It isn't just the advanced PIDF stuff that gets noticed

I am thinking that we should meet Tuesday and review some things as well as discuss our plans to follow. I have some ideas on auton that will make it much closer than what we have been seeing. Await for Ms. Chappell or Glushyn to send out an email to confirm.
